datalad/crawler/nodes/tests/test_misc.py

```diff
@@ -41,12 +44,62 @@
 @skip_if_no_network
 @use_cassette('brain-map.org-1', return_body='')
 def test_get_disposition_filename():
-    input = {'url': 'http://human.brain-map.org/api/v2/well_known_file_download/157722290'}
-    output = list(get_disposition_filename(input))
+    inputted = {'url': 'http://human.brain-map.org/api/v2/well_known_file_download/157722290'}
+    output = list(get_disposition_filename(inputted))
     eq_(len(output), 1)
     eq_(output[0]['filename'], 'T1.nii.gz')
 
 
+def test_get_url_filename():
+    inputted = {'url': 'http://human.brain-map.org/api/v2/well_known_file_download/157722290'}
+    output = list(get_url_filename(inputted))
+    eq_(len(output[0]), 2)
```

Comment:
My intention was to mimic  `test_get_disposition_filename`. Changed to check equality of full output value.